index.html

```diff
@@ -260,7 +263,42 @@ <h1 class="section-title">
                 </div>
             </section>
             <!-- /PAGE ABOUT -->
-
+            <!-- PAGE DIVERSITY -->
+            <section class="page-section" id="diversity">
+                <div class="container">
+                    <h1 class="section-title">
+                        <span data-animation="flipInY" data-animation-delay="300" class="icon-inner"><span
+                                class="fa-stack"><i class="fa rhex fa-stack-2x"></i><i
+                                class="fa fa-star fa-stack-1x"></i></span></span>
+                        <span data-animation="fadeInRight" data-animation-delay="500" class="title-inner">Diversity Matters <small>/ We welcome you</small></span>
+                    </h1>
+                    <div class="row">
+                        <div class="col-lg-8">
```

Comment:
Since we don't have any second column showing photos as in the previous section, we could allow this div to fit all the 12 vertical columns (that is, `col-lg-12` class instead of `col-lg-8`).